docs/dev/metrics.md

```diff
@@ -2,13 +2,23 @@
 
 The Cluster Version Operator reports the following metrics:
 
-Core metrics for the version
+Core metrics for the version. Types:
+
+* `current` - the version the operator is applying right now (the running CVO version)
+* `failure` - if the failure condition is set, reported for desired and current versions
+* `desired` - reported if different from current
+* `completed` - returns 1 and the info of the most recent `Completed` entry in the history, or 0 otherwise
+
+`current` has the `age` label set to the unix timestamp of the payload age, `completed` has the `age` label
```

Comment:
User seeing the metrics current with these others is going to assume that the value has probably something to do with cvo applying it.. while this value is only related to the release image?

Reply:
I agree it's a reasonable concern.  Right now these two things are kind of coupled - I could split them, but it's not clear what the split is.  I could add another label.  Not sure.

Reply:
I could imagine us having `cluster_version_image` and `cluster_version`, with `type` matching across them.  I need to see what the actual data is though